### ISSUE TITLE
Natural Crit Bugs

### ISSUE THREAD

**Issue by cheerio-buffet** [stolksdorf/naturalcrit#59]
**Browser Type/Version**: [Google Chrome Version 48.0.2564.116 m]

**Operating System**: [Windows 10]

**Issue Description**: [1. The borders and dividers on Monster Stat Blocks don't show up on the right column of a page, only the left. Tried with and without a Column Break. (image 1) 2. When going to print pages with Monster Stat Blocks the tables become landscape oriented and run off the page.(image 2)  3. Only the first page will show up in Print View no matter how many pages there actually are. (image 3&4) ]

**Markdown code to reproduce**:  For the Monster Stat Blocks 

```
___
> ## Sneig Hyurgrall
>*Medium half-orc,  chaotic neutral*
> ___
> - **Armor Class** 14 (unarmored defense) 
> - **Hit Points** 24 (2d12)
> - **Speed** 30ft.
>___
>|STR|DEX|CON|INT|WIS|CHA|
>|:---:|:---:|:---:|:---:|:---:|:---:|
>|17 (+3)|14 (+2)|14 (+2)|8 (-1)|10 (0)|12 (+1)|
>___
> - **Senses** passive Perception 10
> - **Languages** Common, Orc, Giant
> - **Challenge** 2 (40 XP)
> ___
> ***Reckless.*** On your first attack of your turn you can gain advantage on all melee weapon attack rolls that use Strength, but attack rolls against you have advantage until the start of his next turn. 

> ### Actions
> ***Great Axe.*** *Melee Weapon Attack:* +5 to hit, reach 5ft., one target. *Hit* 9 (1d12 + 3) slashing damage. 
>
> ***Hand Axe.*** *Melee Weapon Attack:* +5 to hit,reach 5ft., one target. *Hit* 6 (1d6 + 3)


___
> ## Biri Myastan
>*Medium Dragonborn, lawful good*
> ___
> - **Armor Class** 19
> - **Hit Points** 20 (2d10)
> - **Speed** 30ft.
>___
>|STR|DEX|CON|INT|WIS|CHA|
>|:---:|:---:|:---:|:---:|:---:|:---:|
>|17 (+3)|10 (0)|15 (+2)|10 (0)|10 (0)|12 (+1)|
>___
> - **Dragon Ancestry** Gold dragon, fire type
> - **Condition Immunities** Fire resistance
> - **Senses** passive Perception 10
> - **Languages** Common, Draconic
> - **Challenge** 2 (40 XP)
> ___

> ### Actions
> ***Longsword.*** *Melee Weapon Attack:* reach 5ft., one target. *Hit* 7 (1d8 + 3) 
>
> ***Hand Axe.*** *Melee Weapon Attack:* reach 5ft., one target. *Hit* 3 (1d6)
>
> ***Fire Breath.*** *Range Weapon Attack:* range 15ft. cone, multiple targets. **DC 12 Dexterity saving throw** *Failed save:* 2d6 fire damage. *Successful save:* Half damage. 
>___

___
> ## Amnestria Galanodel
>*Medium High Elf, chaotic good*
> ___
> - **Armor Class** 12
> - **Hit Points** 14(2d6+2)
> - **Speed** 30ft.
>___
>|STR|DEX|CON|INT|WIS|CHA|
>|:---:|:---:|:---:|:---:|:---:|:---:|
>|8 (-1)|12+2 (+2)|14 (+2)|15+1 (+3)|14 (+2)|8 (-1)|
>___
> - **Condition Immunities** Cannot be put to sleep, advantage against charm effects
> - **Senses** passive Perception 12
> - **Languages** Common, Elvish, Sylvan, Inferal, Draconic
> - **Challenge** 2 (40 XP)
> ___

> ### Actions
> ***Shortsword.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) piercing damage. 
>
> ***Fire Bolt.*** *Ranged Weapon Attack:* +5 to hit, range 120ft., one target. *Hit* 5 (1d10 fire) damage
>___

___
> ## Tannelia Maggwyn
>*Medium High Elf, neutral evil*
> ___
> - **Armor Class** 12
> - **Hit Points** 32(5d6 + 2)
> - **Speed** 30ft.
>___
>|STR|DEX|CON|INT|WIS|CHA|
>|:---:|:---:|:---:|:---:|:---:|:---:|
>|8 (-1)|14 (+2)|14 (+2)|18 (+4)|14 (+2)|8 (-1)|
>___
> - **Condition Immunities** Advantage against charmed, magic cannot put you to sleep
> - **Senses** passive Perception 15
> - **Languages** Common, Elf, Sylvan, Draconic
> - **Challenge** 3 (700 XP)
> ___

> ### Actions
> ***Shortsword.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) piercing
>
> ***Fire Bolt.*** *Ranged Weapon Attack:* +7 to hit, range 120ft., one target. *Hit* 11 (2d10 fire) damage
>
> ***Ray of Frost.*** *Ranged Weapon Attack:* +7 range 60ft., one target. *Hit* 8 (2d8) Target's speed is reduced by 10 feet until the start of you next turn. 


```

**Related Images** : 
![natural crit bugs 1](https://cloud.githubusercontent.com/assets/17627206/13513121/7962f5c4-e164-11e5-8292-8d1be39d2c59.png)
![natural crit bugs 2](https://cloud.githubusercontent.com/assets/17627206/13513119/795ff75c-e164-11e5-921f-5f3d7b6edf14.png)
![natural crit bugs 3](https://cloud.githubusercontent.com/assets/17627206/13513122/79636a90-e164-11e5-908c-1c5627d930a4.png)
![natural crit bugs 4](https://cloud.githubusercontent.com/assets/17627206/13513120/79624660-e164-11e5-8b8e-2815d588aa40.png)

**Comment by guppy42** [comment]
In regards to 

> 2. When going to print pages with Monster Stat Blocks the tables become landscape oriented and run off the page.

Make sure to use Chrome 49 or later, or alternatively Chrome "canary" - printing columns are not supported before this.

> 3. Only the first page will show up in Print View no matter how many pages there actually are. (image 3&4) ]

Hit ctrl+f5 on the print view - it sometimes lags a little behind

This screenshot was done on Chrome 49 (OS: LDME 2, 64bit)
![Screenshot]
(http://i.imgur.com/rJsvkYH.png)


This first issue has to do with the position:absolute attribute on the :before class, and may not be fixable.

Cool Images btw where are they from?

**Comment by stolksdorf** [comment]
Fixed both issues happening here. 

1) I used pseudo-elements to draw the complex borders, which combined with css columns just killed them. Replaced those with images.
2) Chrome has issues converting blur effects (eg. box shadows) into PDF. So when printing to pdf it removes the box shadows.

Both fixes on in v2, which is up on [staging now](http://natural-crit-staging.herokuapp.com/homebrew/share/rJa2rgBf)
